### DISCUSSION TITLE
failed to load config: ValidateConfig: either bitcoin.active or litecoin.active must be set to 1 (true)

### DISCUSSION THREAD

**Discussion by vicalrez** [lightningnetwork/lnd#7430] [Troubleshooting]
I had the following issues when I tried to run my lightening node on windows 11: failed to load config: ValidateConfig: either bitcoin.active or litecoin.active must be set to 1 (true)

I have the bitcoin core and lightening node in a separate disck.

**Comment by ellemouton** [comment]
Hi @vicalrez , are you setting `bitcoin.active=1` in your config file?

**Reply by vicalrez** [reply]
Yes I did. My config file is not in the default directory. Can it affect the lightning node setting?

**Comment by vicalrez** [comment]
Yes I did it.Thanks for answer.Sent from my Galaxy
-------- Original message --------From: Elle ***@***.***> Date: 2/20/23  12:36 PM  (GMT-05:00) To: lightningnetwork/lnd ***@***.***> Cc: vicalrez ***@***.***>, Mention ***@***.***> Subject: Re: [lightningnetwork/lnd] failed to load config: ValidateConfig:
  either bitcoin.active or litecoin.active must be set to 1 (true) (Discussion
  #7430) 
Hi @vicalrez , are you setting bitcoin.active=1 in your config file?

**Reply by ellemouton** [reply]
and did that solve your problem?

**Reply by vicalrez** [reply]
No it didn't.

**Reply by ellemouton** [reply]
Ok, next question would be: are you sure that the correct config file is being pointed to?

**Reply by vicalrez** [reply]
See below how I write the script for bitcoin.conf:

server=1
daemon=1
bitcoind.dir=D:\Bitcoin\daemon
bitcoind.rpchost=127.0.0.1
bitcoind.rpcuser=**********
bitcoind.rpcpass=******
bitcoind.zmqpubrawblock=tcp://127.0.0.1:28332
bitcoind.zmqpubrawtx=//127.0.0.1:28333
datadir=D:\Bitcoin

**Reply by ellemouton** [reply]
pls see my question above: are you sure that the correct _lnd_ config file is being pointed to when you start lnd? from the error you sent, it looks like it is not reading the file

**Reply by vicalrez** [reply]
yes, The lnd. conf file is inside the lnd folder: c:\users\xxxxx\appdata\local\lnd

**Reply by vicalrez** [reply]
I mean I create the lnd.cof file and located inside the lnd folder.

**Reply by ellemouton** [reply]
when you start lnd, are you definitely pointing it to that config file though? are you using the `--configfile` option?

**Reply by vicalrez** [reply]
no, I ll try to do that. thanks.
